_unittest/test_09_Primitives2D.py

```diff
@@ -74,6 +81,25 @@ def test_06_create_region(self):
         region = self.aedtapp.modeler.create_region([100, 100, 100, 100, 100, 100])
         assert not region
 
+    def test_06_a_create_region_Z(self):
+        if self.axisymmetrical.modeler["Region"]:
+            self.axisymmetrical.modeler.delete("Region")
+        assert "Region" not in self.axisymmetrical.modeler.object_names
+        assert self.axisymmetrical.modeler.create_region([100, 50, 20])
```

Comment:
@IreneWoyna I'd assert after each create_region that your Region object is now in the list of the objects. Before the delete()